config/bidderinfo.go

```diff
@@ -109,6 +112,10 @@ func (s *Syncer) Override(original *Syncer) *Syncer {
 		copy.Redirect = s.Redirect.Override(original.Redirect)
 	}
```

Comment:
I realize this isn't part of your PR but, should we use this opportunity to merge the repetitive `if` statements above into one?
```
103     if original == nil {
104         copy.IFrame = s.IFrame.Override(nil)
    +       copy.Redirect = s.Redirect.Override(nil)
105     } else {
106         copy.IFrame = s.IFrame.Override(original.IFrame)
    +       copy.Redirect = s.Redirect.Override(original.Redirect)
107     }
108
109 -   if original == nil {
110 -       copy.Redirect = s.Redirect.Override(nil)
111 -   } else {
112 -       copy.Redirect = s.Redirect.Override(original.Redirect)
113 -   }
config/bidderinfo.go
```
Or maybe we could correct it in a separate PR and properly test the change there. I'm good either way.

Reply:
Sure. I didn't notice the repetition. Collapsed.